src/app.js

```diff
@@ -21,13 +22,16 @@ export function app(app) {
     })
   }
 
-  if (document.readyState[0] !== "l") {
-    load()
-  } else {
-    addEventListener("DOMContentLoaded", load)
-  }
+  return emit("loaded", emit, render(state, view))
```

Comment:
No, it won't. You need to use `DOMContentLoaded` if you are unsure like if you are loading your app bundle inside the `<head>`. You don't often do that, yes?